ballista/core/src/config.rs

```diff
@@ -87,6 +87,17 @@ impl BallistaConfigBuilder {
         Self { settings }
     }
 
+    pub fn load_env(&self) -> Self {
+        let mut settings = self.settings.clone();
+        if let Ok(it) = env::var("DATAFUSION_CATALOG_LOCATION") {
+            settings.insert("datafusion.catalog.location".to_string(), it);
```

Comment:
The Flight SQL page in the user guide should be updated with information about these env vars, and the configs.md page should be updated with these DataFusion settings.

Reply:
> Flight SQL page in the user guide should be updated

I can do that, but I think it should work across the board - even if you connect from a regular ballista client. I'll test that.

> configs.md page should be updated with these DataFusion settings

Thanks, I was unaware. I'll go do that.